modules/examiner/php/editcertification.class.inc

```diff
@@ -0,0 +1,277 @@
+<?php
+/**
+ * Examiner module
+ *
+ * This processes the values of the Edit Certification form and
+ * inserts updates into the database.
+ *
+ * PHP Version 7
+ *
+ * @category Main
+ * @package  Loris
+ * @author   Zaliqa Rosli <zaliqa.rosli@mcin.ca>
+ * @license  http://www.gnu.org/licenses/gpl-3.0.txt GPLv3
+ * @link     https://github.com/aces/Loris
+ */
+
+namespace LORIS\examiner;
+use \Psr\Http\Message\ServerRequestInterface;
+use \Psr\Http\Message\ResponseInterface;
+
+/**
+ * This processes the values of the Edit Certification form and
+ * inserts updates into the database.
+ *
+ * PHP Version 7
+ *
+ * @category Main
+ * @package  Loris
+ * @author   Zaliqa Rosli <zaliqa.rosli@mcin.ca>
+ * @license  http://www.gnu.org/licenses/gpl-3.0.txt GPLv3
+ * @link     https://github.com/aces/Loris
+ */
+class EditCertification extends \NDB_Page
+{
+    public $skipTemplate = true;
+
+    /**
+     * Checks if the user has access to the edit examiner form
+     *
+     * @return bool
+     */
+    function _hasAccess()
+    {
+        $user = \User::singleton();
+        return $user->hasPermission('examiner_view')
+            && $user->hasPermission('examiner_multisite');
```

Comment:
Should this maybe be an OR permission? I think `multisite` implies that you have `view`. It also means that a user has to be able to edit ALL sites, or none at all